form_validate.js

```diff
@@ -0,0 +1,14 @@
+function validate(event) {
+  event.preventDefault(); // stop form from submitting
+
+  const name = document.getElementById("name").value.trim();
+  const email = document.getElementById("email").value.trim();
+
+  if (name === "" || email === "") {
+    alert("Please enter both your name and email!");
+    return false;
+  }
+
+  alert("Thanks for completing the quiz, " + name + "!");
```

Comment:
You didn't check if we answered all questions.

Or you could add "required" in the html input for all questions.